main.go

```diff
@@ -78,6 +91,13 @@ func main() {
 		collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}),
 	)
 
+	// add metrics for total duration of tenants file reload
```

Comment:
Will you add this now or later?

Reply:
Hum, bad comment. Not planning to do that. Probably forgot to remove that after a change.